### ISSUE TITLE
Run ettercap with filters disabled

### ISSUE THREAD

**Issue by rm1405** [Ettercap/ettercap#1232]
Can I run ettercap from the cli and load a few filters but they should be disabled and I will enable them later?

**Comment by koeppea** [comment]
No. They're right away active and can only be deactivated during the session using the interactive menu.
